docs/changes.rst

```diff
@@ -120,6 +120,34 @@ Released: not yet
 * Made the MOFWBEMConnection class internal and undocumented. It has an
   inconsistent semantics and should not be used by users. (See issue #2001).
 
+* The migration to use the 'requests' Python package for communication between
```

Comment:
We did not remove it as it was never part of the external API. pywbem states since a long time what the external API is (only what is in the pywbem namespace), and that function was never in the external API. If we make an exception for the change to this internal function, do we now log changes to any other internal functions, too?

Reply:
OK